src/app/detail/[task_id]/[user_type]/page.tsx

```diff
@@ -152,6 +154,11 @@ export default async function TaskDetailPage({
                       const data = await getSignedUrlUpload(token, fileName)
                       return data
                     }}
+                    postScrapImage={async (payload: ScrapImageRequest) => {
+                      'use server'
+                      const data = await postScrapImage(token, payload)
+                      return data
+                    }}
```

Comment:
Let's extract this in a variable. And also 159, 160 could be simplified in a single return statement.